src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2370,6 +2390,7 @@ def webapp_up(cmd, name, location=None, sku=None, dryrun=False, logs=False):  #
         _set_build_app_setting = True
         # Configure default logging
         _configure_default_logging(cmd, rg_name, name)
+        # TODO: Set always on to true
```

Comment:
valid comment?

Reply:
yeah I want to add this with the next set of changes - but I will remove & track this as an issue